docs/deploying-airbyte/on-aws-ec2.md

```diff
@@ -63,7 +74,7 @@ ssh -i $SSH_KEY ec2-user@$INSTANCE_IP
 ``` bash
 mkdir airbyte && cd airbyte
 wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,flags.yml,docker-compose.yaml}
-docker compose up -d # run the Docker container
+docker-compose up -d # run the Docker container
```

Comment:
Airbyte needs docker version 2 which includes compose command. Please revert this change.

Reply:
this is docker-compose command, I don't think I understand what you mean by docker version 2 here.